backend/scripts/cases/data_load.py

```diff
@@ -0,0 +1,24 @@
+import csv
+from typing import Dict
+from config import CSV_PARTIES, CSV_DOCUMENTS, CSV_CASES, CSV_EVENTS
+
+
+def _data_load(filename: str, id_key: str) -> Dict[str, Dict]:
```

Comment:
```suggestion
from config import CSV_PARTIES, CSV_DOCUMENTS, CSV_CASES, CSV_EVENTS


def _data_load(filename: str, id_key: str) -> dict[str, dict]:
```

Reply:
It is also nice to consider whether we really want `dict[str, dict]` rather than `Mapping[str, Mapping]`... if we never want to modify the structure directly, using the immutable types really does let pyright/mypy identify when we're being bad :smiling_imp: